Framework/DataHandling/test/NXcanSASTestHelper.h

```diff
@@ -10,15 +10,15 @@
 #include "MantidAPI/MatrixWorkspace_fwd.h"
 #include "MantidAPI/WorkspaceGroup_fwd.h"
 
-#include <Poco/TemporaryFile.h>
+#include <filesystem>
 #include <string>
 #include <vector>
 
 namespace NXcanSASTestHelper {
 struct NXcanSASTestParameters {
   NXcanSASTestParameters() {}
 
-  std::string filename{Poco::TemporaryFile::tempName() + ".h5"};
+  std::string filename{std::filesystem::temp_directory_path().string() + "testFile.h5"};
```

Comment:
```suggestion
  std::string filename{(std::filesystem::temp_directory_path() / "testFile.h5").string()};
```
I would suggest doing it like this to avoid worrying about slashes.